### PR TITLE
[2.3.1] [#2044] update changelog 

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
- [x] changelog is up-to-date for version 2.3.1
Related to #2044 

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
 
+## 2.3.1 / 2019-07-05
+
+### Fixed issues
+
+- Search: #2001, #2016, #2025
+- Docker: #2005, #2054
+- Timeline: #2032
+- Sharedspace: #2058
```
Comment (lebouquetin) [inline changelog.md]: "shared space" (with a space between shared and space)

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+- Admin user: #1470, #2034
```
Comment (lebouquetin) [inline changelog.md]: admin user => user management

File: changelog.md [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+- Email: #2036
+- Preview-generator: #2022
```
Comment (lebouquetin) [inline changelog.md]: preview-generator -> content preview

File: changelog.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

**Pull request merged**